### PR TITLE
Fix war deployment

### PR DESCRIPTION
**Pull request by muff1nman**

### CODE REVIEW

**Comment by muff1nman** [comment]
Any comments/thoughts?

**Comment by biconou** [comment]
Nice. I will test and review it this week end I think.

File: libresonic-main/src/main/java/org/libresonic/player/boot/Application.java [hunk 1]
```diff
@@ -31,19 +39,124 @@ public ServletRegistrationBean dwrServletRegistrationBean() {
...
-        SpringApplication springApplication = new SpringApplication(Application.class);
-        springApplication.addInitializers(new AdditionalPropertySourceConfigurer());
-        springApplication.run(args);
+        new Application().configure(new SpringApplicationBuilder(Application.class))
+                         .web(true)
+                         .initializers(new AdditionalPropertySourceConfigurer())
+                         .run(args);
     }
 
 }
```
Comment (biconou) [inline libresonic-main/src/main/java/org/libresonic/player/boot/Application.java]: God. I realized that I added these lines to my PR branch but missed to push it !!

**Comment by biconou** [comment]
I think that this PR should be merged. 
As I missed to push some filters, especially those related with cache, let's see if it fixes some of the @fxthomas 's performance issues.

**Comment by biconou** [comment]
I just tested both standalone package and war deployed to tomcat 8.
It works !
Nice job @muff1nman

**Pull request merged**